packages/freshchat-api/README.md

```diff
@@ -32,6 +40,23 @@ freshchat.conversationStatusUpdate('<conversation-id>', 'resolved');
 freshchat.conversationStatusUpdate('<conversation-id>', 'new');
 ```
 
+### Generate chat Transcript
+```javascript
 
-
-
+var app_url = 'https://domain.freshchat.com';
+var app_alias = '<App ID>'; //Available from Settings--> Mobile SDKs
+var conversationID = '<Freshchat Conversation UUID>';
+
+freshchat.getConversationTranscript(app_url, app_alias, conversationID, {
+	//The below value can either be 'text' or 'html'
+	output: 'html', 
+	//Below is a Flag to include the conversation link in the generate transcript.
+	isIncludeFreshchatLink: true, 
+	//Below is a Flag to generate transcript for the entire conversation or every interaction. (Create -> Resolve, Reopen -> Resolve)
```

Comment:
Create -> Latest message 
Reopen -> Latest message

This would more accurately depict what it does.